addons/mail/static/src/model/model_manager.js

```diff
@@ -95,7 +95,7 @@ export class ModelManager {
          * mode adds more integrity checks and more verbose error messages, at
          * the cost of performance.
          */
-        this.isDebug = false;
+        this.isDebug = true;
```

Comment:
it's in a `WIP` commit so ok. Just don't forget to reset to `false` before merge